utils/math.h

```diff
@@ -22,6 +22,9 @@
 #ifndef PV_MATH_H
 #define PV_MATH_H
 
+#define MAX_DEC_STRING_SIZE_OF_TYPE(type) \
```

Comment:
how about changing the name of this define to MAX_DIGIT_COUNT_OF_TYPE or similar?